CHANGELOG.md

```diff
@@ -34,6 +34,11 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 ### Bug fixes
 
+- Overlay is blocked on the `panfs` filesystem, allowing sandbox directories to be
+  run from `panfs` without error.
+
```

Comment:
I previously asked you to put this here, but I changed my mind and would like you to instead move this message up under the "Changed defaults/behaviors" section.